pkgs/development/ocaml-modules/ao/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildDunePackage, fetchFromGitHub, dune-configurator, libao }:
+
+buildDunePackage rec {
+  pname = "ao";
+  version = "0.2.4";
+
+  minimalOCamlVersion = "2.0";
```

Comment:
This line is surprising…

Reply:
I agree, can't figure out why I marked it as such I might have misread read dune >= 2.0 as ocaml >= 2.0